app/app.go

```diff
@@ -479,7 +514,9 @@ func NewIxoApp(
 		// Note that since this reads from the store, we can only perform it when
 		// `loadLatest` is set to true.
 		ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
-		app.CapabilityKeeper.InitializeAndSeal(ctx)
+		// TODO: Check if this will work
+		app.CapabilityKeeper.InitMemStore(ctx)
+		app.CapabilityKeeper.Seal()
```

Comment:
@ig-shaun @demondayza @steveixo 
This `app.CapabilityKeeper.InitializeAndSeal(ctx)` was made private, but contained the functions referenced in this comment does this make sense. Is there something else I need to consider here?